src/plugins/dashboard/public/application/lib/sync_dashboard_index_patterns.ts

```diff
@@ -74,7 +82,9 @@ export const syncDashboardIndexPatterns = ({
       distinctUntilChanged(deepEqual),
       switchMap((newChildIds: string[]) =>
         merge(...newChildIds.map((childId) => dashboardContainer!.getChild(childId).getOutput$()))
-      )
+      ),
+      // Embeddables often throw errors into their output streams.
+      catchError(() => EMPTY)
```

Comment:
nit: I'd move this to a specific child's observable instead: 

```
switchMap((newChildIds: string[]) =>
         merge(...newChildIds.map((childId) => dashboardContainer!.getChild(childId).getOutput$().pipe(catchError(() => EMPTY))))
       ),
```

I don't think there is a functional difference in how the whole pipe works, but semantically I think this would make more sense as we want to mute errors from a specific output

Reply:
This does make a bit more sense, I have moved the catchError into the embeddable output streams instead. Thanks!